### DISCUSSION TITLE
Dockets Not Pulling Correctly for insb court

### DISCUSSION THREAD

**Discussion by arharvard** [freelawproject/courtlistener#2820] [Q&A]
Hello,

Not sure if this is the place, but dockets for Indiana Southern Bankruptcy court (insb) are not pulling correctly. It is just pulling partial docket with judge abbreviation (from 2010), and in the core docket it adds a 0 after the year (for all years).

See screen shots for example, along full case.

![CL_example](https://github.com/freelawproject/courtlistener/assets/127220811/0c2e8878-5035-4628-a786-a552759408a2)
![cl](https://github.com/freelawproject/courtlistener/assets/127220811/11315e5c-c362-46f5-9fee-62f90ad7e6f2)
![docket](https://github.com/freelawproject/courtlistener/assets/127220811/2160b1e2-dda7-490d-bead-d0327a853688)

**Comment by mlissner** [comment]
Perfectly good place to raise this, thanks. 

Do you know how many dockets this issue affects or if it affects ones with a recent date_created or date_modified field?

**Comment by arharvard** [comment]
From what I found, it affects all insb from 2010, which is a couple of hundred thousand (200k) at the least (I'm using dockets-2023-03-31).

**Comment by mlissner** [comment]
Cool. We should clean that up, I guess. But just to confirm, it doesn't seem like this is a problem that's affecting any new items, right?

**Comment by arharvard** [comment]
not sure what you mean by new, but all dockets from 2010 until the latest ones are affected by this. Everything before and including 2009 is not affected. see example above, dockets from 2020, 2012, 2017 etc. are affected.

**Reply by mlissner** [reply]
How are you pulling the data that has that situation? When I do a randomly-ordered search on RECAP, I don't see it at all: https://www.courtlistener.com/?type=r&q=&type=r&order_by=random_123%20desc&court=ilcb%20ilnb

**Reply by arharvard** [reply]
The issue arises when for Indiana Southern Bankruptcy (insb) court , where you can also see the issue in the docket number (this link: https://www.courtlistener.com/?type=r&q=&type=r&order_by=random_123%20desc&court=ilcb%20insb), the link you shared shows results for Illinois North Bankruptcy Court. 
From what I saw this specific issue only occurs for Indiana Southern Bankruptcy court.

**Reply by mlissner** [reply]
Hm, @albertisfu, can you see if this is an issue with Juriscraper? Looks like somehow we're screwing up docket numbers in at least one bankruptcy court. Once we know the cause, can you open an issue to fix the cause and to cleanup the data (if we can)?

**Reply by albertisfu** [reply]
Sure! I'm checking this issue.

**Reply by albertisfu** [reply]
Ok, yeah there is a problem parsing docket numbers in INSB, I've created the issue in Juriscraper:
https://github.com/freelawproject/juriscraper/issues/690

But there's an additional issue, some of these dockets don't have a case name, some examples:
https://www.courtlistener.com/docket/54071798//
https://www.courtlistener.com/docket/54700541//
https://www.courtlistener.com/docket/54966602//
https://www.courtlistener.com/docket/54897146//
https://www.courtlistener.com/docket/52969957//

I tried to find the source of these dockets, but I couldn't trace them. There are no PQ objects for these dockets, and they weren't added by the RSS feed since the entries are older than the date when we began scraping feeds.

If one of these dockets is re-uploaded from PACER it gets a case name. So I believe the issue lies in the original source of these dockets.

All these dockets share a common creation date, January 29, 2021. Could there have been a special import on that day? If we still have the original dockets from that import stored somewhere, we could reprocess and correct these dockets.

**Reply by mlissner** [reply]
Were you able to figure out how many dockets had no case name in INSB?

**Reply by albertisfu** [reply]
Yeah, there are around 650,000 dockets with this problem on INSB

Is interesting that most of these dockets were created on December 2020 and January 2021, the day with more dockets created is January 29, 2021.

[insb_blank_case_name_rows.csv.zip](https://github.com/freelawproject/courtlistener/files/11772124/insb_blank_case_name_rows.csv.zip)

**Reply by mlissner** [reply]
That is when we pulled iquery data for every case in the country. I don't think we saved that html, but we could pull it again, I guess.
